tools/internal_ci/linux/grpc_distribtests_ruby.sh

```diff
@@ -46,6 +49,8 @@ cp -r artifacts/ruby_native_gem_*/* input_artifacts/ || true
 # a better signal about which distribtest are affected by the currently broken artifact builds.
 tools/run_tests/task_runner.py -f distribtest linux ruby -j 6 -x distribtests/sponge_log.xml || FAILED="true"
 
+tools/internal_ci/helper_scripts/store_artifacts_from_moved_src_tree.sh
```

Comment:
This is a distribtest, do we need to preserve artifacts?

Reply:
see comment above.